src/doc/src/reference/unstable.md

```diff
@@ -1570,6 +1571,58 @@ implicit-features = "warn"
 workspace = true
 ```
 
+## Path Bases
+
+* Tracking Issue: [#14355](https://github.com/rust-lang/cargo/issues/14355)
```

Comment:
This never makes mention of the unstable feature to activate this.

Reply:
Added `cargo-features` to the example Cargo.toml

Reply:
Thanks!